api/constants/constants.go

```diff
@@ -17,6 +17,54 @@ var (
 	EulaMessage string
 )
 
+const (
+	FullGroupName = "symphony"
+
+	// system annotations, reserved and should not be modified by client.
+	AzureCorrelationIdKey        = "management.azure.com/correlationId"
+	AzureEdgeLocationKey         = "management.azure.com/customLocation"
+	AzureOperationIdKey          = "management.azure.com/operationId"
+	AzureNameIdKey               = "management.azure.com/azureName"
+	AzureResourceIdKey           = "management.azure.com/resourceId"
+	AzureSystemDataKey           = "management.azure.com/systemData"
+	AzureTenantIdKey             = "management.azure.com/tenantId" // Not used
+	RunningAzureCorrelationIdKey = "management.azure.com/runningCorrelationId"
+	SummaryJobIdKey              = "SummaryJobIdKey"
+	OperationStartTimeKeyPostfix = FullGroupName + "/started-at" // instance/target
+
+	ProviderName = "management.azure.com/provider-name"
+)
+
+func SystemReservedAnnotations() []string {
```

Comment:
I'm concerned about Azure specific constants/logics get leaked into OSS. We had this azure feature flag that is designed to be used to compile Azure specific version. Now this is mingled with OSS.

Reply:
I agree with this. But I don't want this to block this PR because this PR also preserve the OSS system managed annotation. @linyguo will open another issue, let's have discussion when handling that issue.

Reply:
Created #584 to follow up this issue